src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -398,4 +458,13 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
             _operators[operator].status = OperatorStatus.DEREGISTERED;
         }
     }
+
+    /// @notice verifies churner's signature on operator churn approval and increments the churner nonce
+    function _verifyChurnerSignatureOnOperatorChurnApproval(bytes32 registeringOperatorId, OperatorKickParam[] memory operatorKickParams, SignatureWithExpiry memory signatureWithExpiry) internal {
+        uint256 churnerNonceMem = churnerNonce;
+        require(signatureWithExpiry.expiry >= block.timestamp, "BLSRegistryCoordinatorWithIndices._verifyChurnerSignatureOnOperatorChurnApproval: churner signature expired");        
```

Comment:
geq/leq looks flipped here -- seems like you are requiring that the signature is expired?

Reply:
we are checking that `expiry >= now`. seems right?

Reply:
oh yeah I'm wrong, ty